interface/globals.php

```diff
@@ -361,11 +361,11 @@ function undoMagicQuotes($array, $topLevel=true) {
  $nav_bg_line = ' bgcolor="#94d6e7" ';
 }
 $login_filler_line = ' bgcolor="#f7f0d5" ';
-$logocode = "<img src='$web_root/sites/" . $_SESSION['site_id'] . "/images/login_logo.gif'>";
+$logocode = "<img class='img-responsive center-block' src='$web_root/sites/" . $_SESSION['site_id'] . "/images/login_logo.gif'>";
 // optimal size for the tiny logo is height 43 width 86 px
 // inside the open emr they will be auto reduced
-$tinylogocode1 = "<img class='tinylogopng' src='$web_root/sites/" . $_SESSION['site_id'] . "/images/logo_1.png'>";
-$tinylogocode2 = "<img class='tinylogopng' src='$web_root/sites/" . $_SESSION['site_id'] . "/images/logo_2.png'>";
+$tinylogocode1 = "<img class='img-responsive' src='$web_root/sites/" . $_SESSION['site_id'] . "/images/logo_1.png'>";
+$tinylogocode2 = "<img class='img-responsive' src='$web_root/sites/" . $_SESSION['site_id'] . "/images/logo_2.png'>";
```

Comment:
Note the tinylogos were added by matrix. These are for 2 small logos which are also shown on the login screen and also within OpenEMR (in the Frames layout) on the top bar. So would ensure the size in both the login screen and the size within OpenEMR on the top bar do not get changed.